server/app/forms/translation/QuestionTranslationForm.java

```diff
@@ -16,19 +16,19 @@ public QuestionTranslationForm() {
     this.questionHelpText = "";
   }
 
-  public String getQuestionText() {
+  public final String getQuestionText() {
     return questionText;
   }
 
-  public void setQuestionText(String questionText) {
+  public final void setQuestionText(String questionText) {
     this.questionText = questionText;
   }
 
-  public String getQuestionHelpText() {
+  public final String getQuestionHelpText() {
     return questionHelpText;
   }
 
-  public void setQuestionHelpText(String questionHelpText) {
+  public final void setQuestionHelpText(String questionHelpText) {
     this.questionHelpText = questionHelpText;
```

Comment:
suggestion (blocking): why mark these methods as final? at first glance it looks like this is inconsistent with the overarching style of the codebase

Reply:
This class allows subclassing. Marking the methods as final indicates that they can't be overridden in a subclass. This is mostly an extension of the "keep things closed until they need to be opened" principle.